spec/lib/puppet_spec/compiler.rb

```diff
@@ -7,8 +7,8 @@ def compile_to_catalog(string, node = Puppet::Node.new('foonode'))
     Puppet::Parser::Compiler.compile(node).filter { |r| r.virtual? }
   end
 
-  def compile_to_ral(manifest)
-    catalog = compile_to_catalog(manifest)
+  def compile_to_ral(manifest, node = Puppet::Node.new('foonode'))
```

Comment:
"foonode" does not seem to be a very good name here, was it selected because of existing tests using that name everywhere?